local-volume/provisioner/deployment/kubernetes/example/default_example_provisioner_generated.yaml

```diff
@@ -8,12 +8,11 @@ metadata:
   namespace: default 
 data:
   storageClassMap: |     
-    fast-disks:
-       hostDir: /mnt/fast-disks
-       mountDir:  /mnt/fast-disks 
+    local-scsi:
```

Comment:
This file should reflect the default when you run helm.  But these settings look gce/gke specific.

Reply:
I used the example helm command from here: https://github.com/kubernetes-incubator/external-storage/tree/master/local-volume/helm#advanced-options

What command would you like me to use?

Reply:
The one in the main README: 
```
helm template ./helm/provisioner > ./provisioner/deployment/kubernetes/provisioner_generated.yaml
```

Reply:
Updated.